### PR TITLE
CoC Spanish translation

### PR DESCRIPTION
**Pull request by Carlaplzgnz**
Hi! I translated this into Spanish.  I'm not sure whether I left the correct spacing. Please, review it and tell me whether I need to change something.

Thanks!

#13

### CODE REVIEW

**Comment by sanscontext** [comment]
Merging!  Thank you so much @Carlaplzgnz !  :D

File: code-of-conduct-ES.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+##Introducción
+
+Nos dedicamos a proporcionar una experiencia libre de acoso para todo el mundo, y no toleramos ningún tipo de vejación entre nuestros colaboradores. Te pedimos que seas considerado y te comportes con profesionalidad y respeto hacia los demás participantes. Esta normativa y procedimientos relacionados se aplican igualmente a cualquier tipo de comportamiento inaceptable que pueda ocurrir fuera del ámbito de las actividades de nuestra comunidad, en todos los espacios comunes -online y en persona- ,así como en la comunicación one-to- one, y en cualquier contexto ya que ese es un comportamiento que puede afectar desfavorablemente a la seguridad y el bienestar de los miembros de la comunidad. Expositores, ponentes, patrocinadores, empleados y otros asistentes a eventos organizados por Docker, Inc (Docker Con, quedadas, grupos de usuarios) u organizados en las instalaciones deben atenerse a estas directrices y normas de conducta.
```
Comment (amegianeg) [inline code-of-conduct-ES.md]: `one-to-(space)one`

File: code-of-conduct-ES.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+
+La diversidad y la integración refuerzan a la comunidad Docker. Potenciamos la participación de la procedencia más variada y diversa posible y queremos ser claros con nuestra posición.
+
+Nuestro objetivo es preservar una comunidad Docker segura y amable para todo el mundo, con independencia de la experiencia, expresión e identidad de género, orientación sexual, discapacidad, apariencia física , talla corporal, raza, etnia, edad, religión, nacionalidad o cualquier otra categoría protegida bajo la ley aplicable.
```
Comment (amegianeg) [inline code-of-conduct-ES.md]: ` , ` (double spaces)

File: code-of-conduct-ES.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- Contacto físico indeseado e inapropiado
+- Intimidación o bullying
+- Atención sexual no deseadaa
+
+##Información y ejecución
+
+- Si eres sujeto o testigo de alguna violación de este código de conducta, por favor contáctanos presentando un informe de incidencias[incident report](https://docs.google.com/forms/d/e/1FAIpQLScezna1ZXRPzC_phSDoPEF4c5nvw8yQW-vvtI8xHjv-BB9MOg/viewform?c=0&w=1), o enviando un correo electrónico a dockercon@docker.com
+- Si fuera necesario, el personal de conferencias está capacitado para llevar acabo las acciones necesarias, las cuales pueden incluir, pero no limitarse a: advertencias, expulsión de la conferencia sin posibilidad de reembolso, y remisión a la seguridad de los locales y aplicación de la ley.
+
+Texto basado en [Slack Developer Community Code of Conduct](https://api.slack.com/docs/community-code-of-conduct), [The Ada Initiative](https://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/), [geekfeminism.org](https://geekfeminism.org/about/code-of-conduct/) e [Drupal Events Code of Conduct](https://events.drupal.org/dublin2016/code-conduct)
```
Comment (amegianeg) [inline code-of-conduct-ES.md]: `conduct/) e [Drupal` -> `conduct/) y [Drupal`

**Pull request merged**